tests/data/notebooks/outputs/ipynb_to_marimo/plotly_graphs.py

```diff
@@ -0,0 +1,51 @@
+import marimo
+
+__generated_with = "0.17.0"
+app = marimo.App()
+
+
+@app.cell(hide_code=True)
+def _(mo):
+    mo.md(r"""This notebook contains complex outputs, including plotly javascript graphs.""")
+    return
+
+
+@app.cell(hide_code=True)
+def _(mo):
+    mo.md(r"""# Interactive plots""")
+    return
+
+
+@app.cell(hide_code=True)
+def _(mo):
+    mo.md(r"""We use Plotly's connected mode to make the notebook lighter - when connected, the notebook downloads the `plotly.js` library from the web.""")
```

Comment:
This might be an edge case in an upcoming PR where I'm trying to stabalize markdown formatting. If `len(line) > MAX_LINE_LENGTH` then

```
mo.md(
    {tag}{quote}{body}{quote}
)
```
---

But it's not merged